functions/ingestion-worker-lambda/tests/test_transformer.py

```
@@ -0,0 +1,80 @@
+import sys, os
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
+from transformer import notion_blocks_to_markdown, notion_page_to_markdown
+
+
+def _block(btype: str, text: str, extra: dict = None) -> dict:
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)